.github/workflows/publish.yaml

```diff
@@ -0,0 +1,20 @@
+on:
+  workflow_run:
+    workflows: [Test]
+    branches: [main]
+    types: [completed]
+name: Publish Snap
+jobs:
+  build:
+    if: ${{ github.event.workflow_run.event == 'push' && github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: snapcore/action-build@v1
+      id: build
+    - uses: snapcore/action-publish@v1
+      env:
+        SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.STORE_LOGIN }}
+      with:
+        snap: ${{ steps.build.outputs.snap }}
+        release: stable
```

Comment:
can we publish to edge instead of stable so that we get a chance to test a build before it gets released?

Reply:
Done